python/rpdk/java/templates/generate/HandlerWrapper.java

```diff
@@ -62,11 +67,10 @@ private void initialiseHandlers() {
 
     @Override
     public ProgressEvent<{{ pojo_name }}, CallbackContext> invokeHandler(
-                final AmazonWebServicesClientProxy proxy,
-                final ResourceHandlerRequest<{{ pojo_name }}> request,
-                final Action action,
-                final CallbackContext callbackContext) {
-
+            final AmazonWebServicesClientProxy proxy,
```

Comment:
is only format change here?

Reply:
Yes. Just changed it to match the indentation level of other methods